### PR TITLE
Add base files for the branch (.travis.yml, .coveragerc, .gitignore, README.md, LICENSE)

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

**Comment by guewen** [comment]
For the travis config file, I could find where are some dependencies, such as `base_external_referentials`.

**Comment by bwrsandman** [comment]
@guewen found it here http://bazaar.launchpad.net/~openerp-commiter/openobject-addons/extra-6.0/files/head:/base_external_referentials/

**Comment by guewen** [comment]
@bwrsandman thanks. I don't think that we should accept modules having dependencies that are not in OCA. On the other hand, the modules are already there...

**Comment by bwrsandman** [comment]
We could adopt the dependancy module, I guess.

**Comment by guewen** [comment]
I'm not sure I want to.

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[report]
```
Comment (bwrsandman) [inline .coveragerc]: With the current travis setup, we don't need to add coveragerc

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+language: python
```
Comment (bwrsandman) [inline .travis.yml:1]: This needs to updated with the new quality tools samples

**Comment by pedrobaeza** [comment]
@guewen, please fix commented issues to go on with this PR.

**Comment by LeartS** [comment]
What's the status on this? Is it still needed? I see those files are present now..

**Comment by guewen** [comment]
@LeartS it was for 6.1. Since the files have changed a lot I close this PR. Anyway the builds can't be green because most of the modules depends on not-oca addons.

**Pull request closed without merging**